http.gemspec

```diff
@@ -19,7 +19,9 @@ Gem::Specification.new do |gem|
   gem.licenses      = ["MIT"]
 
   gem.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
-  gem.files         = `git ls-files`.split("\n")
+  gem.files         = Dir["lib/**/*.rb"] +
+                      %w[http.gemspec README.md CHANGES.md LICENSE.txt]
+
   gem.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
```

Comment:
In fact it worth to group these two:

``` ruby
Dir.chdir(__dir__) do
  git_files = `git ls-files -z`.split("\x0")

  gem.files = git_files.grep(%r{^lib/}) +
    %w[http.gemspec README.md CHANGES.md LICENSE.txt]

  gem.test_files = git_files.grep(%r{^spec/})
end
```

Reply:
But that would include the test files in the binary again, no? At least it does when I use this with `gem build http.gemspec`... not that it makes much difference to the size, I guess just removing `logo.png` at ~8KB from the binary makes all the difference that matters.

Reply:
Distro maintainers who repackage RubyGems as distro-specific packages (e.g. deb, rpm) prefer the tests are included so they can run them to ensure they properly repackaged the code.